doc/ca65.sgml

```diff
@@ -1278,6 +1278,35 @@ writable.
 Pseudo functions expect their arguments in parenthesis, and they have a result,
 either a string or an expression.
 
+<sect1><tt>.ADDRSIZE</tt><label id=".ADDRSIZE"><p>
+
+  The <tt/.ADDRSIZE/ function is used to return the interal address size 
+  associated with a symbol. This can be helpful in macros when knowing the address 
+  size of symbol can allow for custom instructions.
+
+  Example:
+
+  <tscreen><verb>
+        .macro myLDA foo
+
+  	        .if .ADDRSIZE(foo) = 1
+  	          	        ;do custom command based on zeropage addressing:
+  	          	        .byte .A5h, foo
+  	        .elseif .ADDRSIZE(foo) = 2
+  	          	        ;do custom command based on absolute addressing:
+  	          	        .byte .ADh, foo
```

Comment:
What are `.A5h` and `.ADh` supposed to be -- ca65 rejects them.  (If you want them to be hex numbers, then the period must be replaced with a zero.)